src/components/PharmacyCard/PharmacyCard.tsx

```diff
@@ -25,13 +31,30 @@ interface PharmacyProps {
 export function PharmacyCard(props: PharmacyProps) {
   const [booking, setBooking] = useState(props.booking);
   const [lastUpdated, setlastUpdated] = useState(props.lastUpdated);
+  const [showMap, setShowMap] = useState(false);
```

Comment:
Minor nit, but bools are usually name via a prefix convention - there is differing opinions on the matter, but `is`, `can`, `has` are common. In this case I would call this `shouldShowMap` or `isMapShown`